woocommerce/woocommerce-gutenberg-products-block
# Configure link for the proceed to checkout button

As a follow up to https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/1926, this implements a way to select the checkout page from the cart block. This option is only shown if the current page is _not_ the cart page set in settings (or if the option was previously set).

This also implements the notice to inform the user that the page is not set as the default, to match checkout.

![Screenshot 2020-03-16 at 17 24 43](https://user-images.githubusercontent.com/90977/76784251-35543100-67ab-11ea-94c0-8e1dfaad82fb.png)

### How to test the changes in this Pull Request:

1. Edit the cart block page
2. Confirm the feedback notice and page notice is shown correctly
3. Try setting the button option to point to a different page and test frontend
